.github/workflows/ci.yml

```diff
@@ -78,12 +76,21 @@ jobs:
     name: 📊 Q&A
     # All linux jobs must run the same ubuntu version to avoid Rust caching issues !
     runs-on: ubuntu-20.04
+    # We skip _parsec.so compilation given Mypy only cares about _parsec.pyi
+    env:
+      : no_build
```

Comment:
something is wrong I can feel it 9d9bb672d4e47dd95650893aaf16d3b25a3ed5ee

@ADcorpo